src/main/java/cloud/fogbow/cli/utils/CommandUtil.java

```diff
@@ -4,23 +4,74 @@
 import cloud.fogbow.cli.exceptions.FogbowCLIException;
 
 import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
 
 public class CommandUtil {
+
     public static String getSystemUserToken(String systemUserToken, String systemUserTokenPath) throws FogbowCLIException {
-        String actualSystemUserToken = null;
-        if (actualSystemUserToken == null || actualSystemUserToken.isEmpty()) {
-            if (systemUserToken != null && !systemUserToken.isEmpty()) {
-                actualSystemUserToken = systemUserToken;
-            } else if (systemUserTokenPath != null && !systemUserTokenPath.isEmpty()) {
+        return getFileContent(systemUserToken, systemUserTokenPath, Messages.Exception.NO_SYSTEM_USER_TOKEN).replace("\n", "");
+    }
+
+    public static String getApplicationPublicKey(String publicKeyValue, String publicKeyPath) throws FogbowCLIException {
+        return getFileContent(publicKeyValue, publicKeyPath, Messages.Exception.NO_PUBLIC_KEY_PROVIDED).replace("\n", "");
+    }
+
+    public static String getFileContent(String providedResourcePath, String NoResourceFoundException) throws FogbowCLIException {
+        return getFileContent(null, providedResourcePath, NoResourceFoundException);
+    }
+
+    public static String getFileContent(String providedRawValue, String providedResourcePath, String NoResourceFoundException) throws FogbowCLIException {
```

Comment:
The last parameter doesn't follow the naming conventions nor it is a meaningful variable name, can you change it to something like `errorMessage`?